### PR TITLE
Ports Thanos Car from AuStation

### PR DESCRIPTION
**Pull request by MCterra10**
## About The Pull Request
Ports thanos car from austation, sprites by myself

## Why It's Good For The Game
Fun new (dead) meme content

## Changelog
:cl:
add: THANOS CAR THANOS CAR
/:cl:

### CODE REVIEW

**Comment by carshalash** [comment]
Mate, where's the fucking picture of the car?

**Comment by MCterra10** [comment]
fine

**Comment by MCterra10** [comment]
thanos_b
thanos_f
thanos_l
thanos_r

**Comment by mrhugo13** [comment]
Tired of security, i am.
Run them over with my 2018 Thanos Car Civic, i must.
Restore balance to the station, i will.

**Comment by actioninja** [comment]
those sprites are a pretty big bruh moment

**Comment by MCterra10** [comment]
@actioninja I agree, but note that this was literally the first time I had ever opened aseprite or even sprited in general. Also by now you should know that my style of ironic humor here is making PRs that look legit but underneath are actually memes

**Comment by tralezab** [comment]
if this was a real good looking thanos car i would like it...

**Comment by MCterra10** [comment]
@tralezab this is a meme PR and I doubt it will be merged. also feel free to make better sprites

**Comment by Abductkaz** [comment]
new 40TC clown item POG

**Comment by Shadowflame909** [comment]
Just make it an Administration item like the speedwagon. 

The majority of people like more event items/prize items that are rarely seen

**Comment by wesoda25** [comment]
you're bad at meme prs bc you still attempt to sound reasonable in them

File: code/modules/vehicles/cars/thanoscar.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/obj/vehicle/sealed/car/thanoscar
+	name = "thanos car"
+	desc = "THANOS CAR! THANOS CAR!"
```
Comment (PKPenguin321) [inline code/modules/vehicles/cars/thanoscar.dm]: `THANOS CAR\nTHANOS CAR`

**Comment by AnturK** [comment]
*snap*

**Comment by Shadowflame909** [comment]
quick make another thanos meme PR as that'll be the half that survives the merge

**Comment by MCterra10** [comment]
@Shadowflame909 it literally is admin spawn only, where did you get the idea that it's available to players

**Pull request closed without merging**